model_compression_toolkit/pytorch/quantization_facade.py

```diff
@@ -31,17 +32,18 @@
 if importlib.util.find_spec("torch") is not None:
     from model_compression_toolkit.pytorch.default_framework_info import DEFAULT_PYTORCH_INFO
     from model_compression_toolkit.pytorch.pytorch_implementation import PytorchImplementation
+    from model_compression_toolkit.hardware_models.pytorch_hardware_model.pytorch_default import PYTORCH_DEFAULT_MODEL
```

Comment:
As a TensorFlow use the get of the DEFAULT_MODEL

Reply:
fixed